bundle/manifests/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -252,6 +329,9 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
+                env:
+                - name: GIT_REPO_PATH
+                  value: /git
                 image: quay.io/opdev/operator-certification-operator:latest
```

Comment:
This would need to be a `sha` to pass certification. If we have the rbac proxy image somewhere that would need to be a sha as well, and put the proper one in slack.

Reply:
updated. lmk if you catch anything else